boilerplate/app/components/Button.tsx

```diff
@@ -65,6 +65,15 @@ export interface ButtonProps extends PressableProps {
    * Children components.
    */
   children?: React.ReactNode
+  /**
+   * disabled prop, accessed directly for declarative styling reasons.
+   * https://reactnative.dev/docs/pressable#disabled
+   */
+  disabled?: boolean
+  /**
+   * An optional style override for the disabled state
+   */
+  disabledStyle?: StyleProp<ViewStyle>
```

Comment:
If we're visually affecting one part of the component for an interaction, we should probably provide the user with the option to configure all part. Let's add a `disabledTextStyle`. To the props.

Reply:
Ah yes, makes sense!

Should be good in https://github.com/infinitered/ignite/pull/2593/commits/a880064758b7d01b47848b12a160b2bfe5e2b231

I also noticed my disabled style diverged from the `$[something]StyleOverride` pattern, so I refactored to match y'all a little more closely in https://github.com/infinitered/ignite/pull/2593/commits/3b99aca24920c0800f19ac37e9cd2bb29f4dc2f3